packages/components/src/components/time-picker/_time-picker.scss

```diff
@@ -64,6 +64,21 @@
       color: $text-05;
     }
   }
+
+  .#{$prefix}--time-picker--light .#{$prefix}--time-picker__select {
+    background-color: $field-02;
+
+    &:hover {
+      background-color: $hover-ui;
+    }
+
+    &:disabled,
```

Comment:
Also, it seems like `disabled` is not being passed down to the `select` element

Reply:
there is a separate prop for disabling the selects in storybook. should I combine them?

Reply:
Oh, I did not realize that! Up to you